src/cli.ts

```diff
@@ -103,6 +103,10 @@ const arrayToEnum = (enums: string[], enumName: string) => {
     name = name.includes(enumSuffix) ? name : `${name}${enumSuffix}`; // TODO: better checking of suffix existence at the end of name
   }
 
+  if (name.includes(".")) {
+    name = name.replace(/\./g, "_");
```

Comment:
Looks ok. But now i'm wondering about two things:
1. During normalization process enums with the `.` are replaced with `__` (double underscore) so maybe we should treat exported enum names in the same manner?
2. And if the first is true, maybe using `replacers` like in line 66-70 would be a good approach here.

Reply:
nice suggestion..
i would suggest for maybe in future we do custom replacer as well.